docs/src/osdk/reference/manifest.md

```diff
@@ -32,58 +32,103 @@ Below, you will find a comprehensive version of
 the available configuration options in the manifest.
 
 ```toml
-kcmd_args = ["init=/bin/busybox", "path=/usr/local/bin"] # <1>
-init_args = ["sh", "-l"] # <2>
-initramfs="./build/initramfs.cpio.gz" # <3>
-
-[boot]
-loader = "grub" # <4>
-protocol = "multiboot2" # <5>
-grub-mkrescue = "/usr/bin/grub-mkrescue" # <6>
-ovmf = "/usr/bin/ovmf" # <7>
-
-[qemu]
-path = "/usr/bin/qemu-system-x86_64" # <8>
-machine = "q35" # <9>
-args = [ # <10>
-    "-enable-kvm",
-    "-m 2G", 
-    "-device virtio-keyboard-pci,disable-legacy=on,disable-modern=off"
-] 
+[project] # <1>
+type = "kernel"
+
+[run] # <2>
+kcmd_args = ["init=/bin/busybox", "path=/usr/local/bin"] # <3>
+init_args = ["sh", "-l"] # <4>
+initramfs="./build/initramfs.cpio.gz" # <5>
+bootloader = "grub" # <6>
+boot_protocol = "multiboot2" # <7>
+grub-mkrescue = "/usr/bin/grub-mkrescue" # <8>
+ovmf = "/usr/bin/ovmf" # <9>
+qemu_path = "/usr/bin/qemu-system-x86_64" # <10>
+qemu_args = [ # <11>
+    "-machine q35,kernel-irqchip=split",
```

Comment:
I still think grouping these lines under the name `qemu` is a good idea.

Reply:
Well I can agree at this point. But the other grouping suggestions seems weird.